docs/pages/_includes/services/uninstall.md

```diff
@@ -14,6 +14,85 @@ For example, to uninstall the {{ include.data.techName }} instance named `{{ inc
 dcos package uninstall --app-id={{ include.data.serviceName }}-dev {{ include.data.packageName }}
 ```
 
+#### Uninstall flow
+
+Uninstalling the service consists of the following steps:
+- The scheduler is relaunched in Marathon with the environment variable `SDK_UNINSTALL` set to "true". This puts the Scheduler in an uninstall mode.
+- The scheduler performs the uninstall with the following actions:
+  - All running tasks for the service are terminated so that Mesos will reoffer their resources.
+  - As the task resources are offered by Mesos, they are unreserved by the scheduler.
+    - **Warning**: Any data stored in reserved disk resources will be irretrievably lost.
+  - Once all known resources have been unreserved, the scheduler's persistent state in ZooKeeper is deleted.
+- The cluster automatically removes the scheduler task once it advertises the completion of the uninstall process.
+
+Note that once the uninstall operation has begun, it cannot be cancelled because it can leave the service in an uncertain, half-destroyed state.
+
+#### Debugging an uninstall
+
+In the vast majority of cases, this uninstall process goes off without a hitch. However in certain situations there can be snags along the way. For example, perhaps a machine in the cluster has permanently gone away, and the service being uninstalled had some resources allocated on that machine. This can result in the uninstall being stuck, because that machine's resources never being offered to the uninstalling scheduler. In turn, the uninstalling scheduler will not be able to successfully the resources it has on that machine.
```

Comment:
"However, in certain situations, there can be snags along the way"

Reply:
"This can result in the uninstall becoming stuck because Mesos will never offers those resources to the uninstalling scheduler. As such, the uninstalling scheduler will not be able to successfully unreserve the resources it had reserved on that machine."

Reply:
Done